peps/pep-0741.rst

```diff
@@ -1053,6 +1053,89 @@ implemented in Python 3.8.
 Disallow setting the ``"configure_locale"`` option has similar issues.
 
 
+Multi-phase initialization (similar to PEP 432)
+-----------------------------------------------
+
+`Eric Snow proposed
+<https://discuss.python.org/t/pep-741-python-configuration-c-api-second-version/45403/27>`_
+to have multiple "phases" for the initalization, not only the
+"preinitialization" and the "initialization". One advantage is to be
+able to configure Python using Python code rather than C code for last
+phases. See :pep:`432` which proposed a similar concept of multiple
+phases, whereas :pep:`587` was implemented only with the
+preinitialization and the initialization.
```

Comment:
The ability to use Python objects for configuration isn't actually something I care about that much.  My interest in phases is more about making concrete the conceptual divisions of initialization.